lib/index.js

```diff
@@ -13,7 +13,15 @@ var validateSortedUnique = util.validateSortedUnique;
 var ARBITRARY_START_OF_TIME = 0;
 var DEFAULT_DELEGATE = {
   unlink: function(inputPath, outputPath, relativePath) {
-    fs.unlinkSync(outputPath);
+    try {
+      fs.unlinkSync(outputPath);
+    } catch (e) {
+      // QUICKFIX: gracefully handle unexpected unlinks (from broccoli-dependency-funnel)
+      if (typeof e === 'object' && e !== null && e.code = 'ENOENT') {
```

Comment:
This seems slightly incorrect, I believe you mean to not assign the `e.code` property?

```js
if (typeof e === 'object' && e !== null && e.code === 'ENOENT') { return; }
```

Reply:
This is what is failing the CI build.